### DISCUSSION TITLE
Have Jmusicbot accept requests from other bots

### DISCUSSION THREAD

**Discussion by Phosphyr** [jagrosh/MusicBot#1012] [Questions] [answered]
I'm working on a bot that plays a song when I press a button, everything works fine up to and including the bot sending "-play <xyz>" in chat which MusicBot doesn't respond to, I'm assuming due to the request coming from another bot. Is there a way to tell MusicBot to respond to bot requests?

**Comment by jagrosh** [comment, marked as answer]
Bots shouldn't respond or react to other bots, JMusicBot included. You're already writing your own bot; just make that bot play the music too, it doesn't sound like you need JMusicBot for this.

**Reply by Phosphyr** [reply]
Thanks, that's what I ended up doing. Love the bot btw.
